packages/cli/src/lib/check-lockfile.ts

```diff
@@ -33,6 +33,7 @@ function missingLockfileWarning(shouldExit: boolean) {
 
 function multipleLockfilesWarning(lockfiles: Lockfile[], shouldExit: boolean) {
   const packageManagers = {
+    'bun.lockb': 'bun',
```

Comment:
🔥

Reply:
`Lockfile` now includes `bun.lockb` ([here](https://github.com/Shopify/cli/blob/9d4c411176e13edcf352e4a4d5341aedaf1ee476/packages/cli-kit/src/public/node/node-package-manager.ts#L30)) so we'd get a Typescript error without it. If we don't want bun, we need to handle the case where the lockfile isn't a key in `packageManagers` by adding a check before accessing it.

Reply:
Yes yes, I understand we need the changes, and I'm quite excited to support Bun actually 😄

Reply:
ah ok I misunderstood the burn comment haha